### ISSUE TITLE
i am not feeling the smoothness in cyberpunk.

### ISSUE THREAD

**Issue by juhannnnnn** [Nukem9/dlssg-to-fsr3#384]
i have HAGS turned on. i do try to go from 30 to 60 fps but it doesnt feel smoother than the base 30 fps( i know the base fps recommended by amd is 60 fps but still...)

**Comment by ForthtuN** [comment]
Do you have any type of game filter on or using HDR and have saturation on? Any type of filter or after fxs will make the performance really bad, even if saying more frames its often just duplicating if any of those are true

**Comment by juhannnnnn** [comment]
i use the nvidia overlay

> Do you have any type of game filter on or using HDR and have saturation on? Any type of filter or after fxs will make the performance really bad, even if saying more frames its often just duplicating if any of those are true

**Comment by juhannnnnn** [comment]
> Do you have any type of game filter on or using HDR and have saturation on? Any type of filter or after fxs will make the performance really bad, even if saying more frames its often just duplicating if any of those are true

Hello??

**Comment by juhannnnnn** [comment]
> > > Do you have any type of game filter on or using HDR and have saturation on? Any type of filter or after fxs will make the performance really bad, even if saying more frames its often just duplicating if any of those are true
> > 
> > 
> > Hello??
> 
> I am so sick an tired of seeing people posting issues with NO INFO !
> 
> Seriously the first thing you would mention is WHAT HARDWARE you own to even have 1% chance of identifying your expected performance and what generated frames will actually do for you. If you are not getting at least 40 FPS then FORGET about FG without big latency. I do not have any issue with this mod and actually it runs better then native FSR3 on Nvidia hardware. DLSS + FSR3 FG is simply a match made in heaven.

i have i7 10700k and rtx 3070 with 32 gb ram

**Comment by morgan-blondellet** [comment]
yeah you should lower your graphics settings to have an higher base framerate otherwise you will not feel the smoothness of added frames. Even if you get more frames, you get the latency of your base framerate.
